src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -303,6 +301,7 @@ public void setPeriod(String period) {
     public int[] getDefaultJavadocTokens() {
         return new int[] {
             JavadocTokenTypes.JAVADOC,
```

Comment:
it is not clear for me why we need to check javadoc token? This check is for summary and javadoc_inline_tag is enough, isn't it?

Reply:
Summary of a Javadoc can exist with and without `@summary` [eg](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#firstsentence). We need `JavadocTokenTypes.JAVADOC` for javadocs not containing `@summary`